packages/melody-logger/src/index.js

```diff
@@ -0,0 +1,32 @@
+/*eslint no-console: [0,{ allow: ["warn"] }] */
```

Comment:
Not sure if this should be a new package. In case we decide it is better to make it a new package, than we need to move `error` method in `melody-compiler/state` to here as well. Otherwise we should add warn inside to the compiler. Personally moving this out of Compiler and allowing customization via loader is better IMHO.  

What are your opinions about this @pago , @jamesb3ll ?

--
@ayusharma I would wait for the discussion until another change on this case to avoid unnecessary work :)

Reply:
I prefer to keep things simple for as long as they can be simple so I wouldn't go for a new package. I would also not go for a class but just something like `{ error: console.error.bind(console), warn: console.warn.bind(console) }` passed in through the `options` of the plugin API with the default specified here: https://github.com/trivago/melody/blob/master/packages/melody-compiler/src/state/State.js#L28